keps/sig-scheduling/2249-pod-affinity-namespace-selector/README.md

```diff
@@ -251,27 +259,58 @@ constraints across a large number of namespaces. The initial implementation
 of pod affinity/anti-affinity suffered from performance challenges, however 
 over the releases 1.17 - 1.20 we significantly improved that. 
 
-We currently have integration test and clusterloader benchmarks that evaluate the extreme
+We currently have integration and clusterloader benchmarks that evaluate the extreme
 cases of all pods having affinity/anti-affinity constraints to/against each other.
 Those benchmarks show that the scheduler is able to achieve maximum throughput 
 (i.e., the api-server qps limit).
 
 #### Security/Abuse
-see previous discussion [here](https://github.com/kubernetes/kubernetes/issues/68827#issuecomment-468470598)
+See previous discussion [here](https://github.com/kubernetes/kubernetes/issues/68827#issuecomment-468470598)
 NamespaceSelector will allow users to select all namespaces. This may cause a security 
 concern: a pod with anti-affinity constraint can block pods from all other namespaces 
 from getting scheduled in a failure domain.
 
-#### External Dependencies
-We are aware of two k8s projects that could be impacted by this change, the descheduler 
-and cluster autoscaler. Cluster autoscaler should automatically consume the change
-since it imports the scheduler code, the descheduler however doesn't and needs to 
-be changed to take this feature into account. We will open an issue to inform the project
-about this update.
+We will address this concern by introducing a new quota scope named `CrossNamespaceAffinity`
+that admins can use to limit which namespaces are allowed to have pods with affinity terms
+that set the existing `namespaces` field or the proposed one `namespaceSelector`. 
+
+To configure CrossNamespaceAffinity as a limited resource by default (i.e. namespaces are required
+to have a quota object to be able to have pods with cross-namespace affinity), admins could 
+set the kube-apiserver flag --admission-control-config-file to the path of the following
+configuration file:
+
+```go
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: "ResourceQuota"
+  configuration:
+    apiVersion: apiserver.config.k8s.io/v1
+    kind: ResourceQuotaConfiguration
+    limitedResources:
+    - resource: pods
+      matchScopes:
+      - scopeName: CrossNamespaceAffinity
+```
+
+If admins want allow that by default, but disable it for specific namespaces, then they
```

Comment:
what is the mechanism to disallow by default but only allow for specific namespaces?

Reply:
the one described above, configuring it as a limited resource by default.

Reply:
rephrased to hopefully make it clearer